src/PrestaShopBundle/Resources/views/Admin/Improve/International/Locations/Country/Blocks/available_fields_content.html.twig

```diff
@@ -0,0 +1,32 @@
+{% set tabActive = 'active show' %}
+<ul class="nav nav-tabs address-format-tabs" id="custom-address-fields">
+  {% for key, rows in availableFields %}
+    {% if loop.index != 1 %}
+      {% set tabActive = '' %}
+    {% endif %}
+    <li class="nav-item">
+      <a href="#availableListFieldsFor_{{ key }}" class="nav-link  {{ tabActive }}">
```

Comment:
```suggestion
      <a href="#availableListFieldsFor_{{ key }}" class="nav-link  {{ loop.index != 1 ? '' : 'active show' }}">
```

Reply:
very nice review comments :slightly_frowning_face:  had only two situations where I was not sure If should implement the suggestions.